hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/UiProperties.java

```diff
@@ -39,6 +42,41 @@ public void setGravatar(final boolean gravatar) {
         this.gravatar = gravatar;
     }
 
+
+
+    /**
+     * Localization information
+     */
+    public static class Localization implements Serializable {
+        private static final long serialVersionUID = 1L;
+
+        /**
+         * Default localization
+         */
+        private String defaultLocal;
```

Comment:
What about `default` only? Looks better `uiProperties.getLocalization().getDefault()`. You could also set the default value here so you don't have to specify the default within `application properties`

Reply:
`default` is a keyword and can't be used as variable, or can it?

Reply:
Yes indeed, sometimes Java is doing things I don't expect. Method `getDefault()` is ok, but `private String default` not

Reply:
Would it be acceptable to name the getter different than the variable?

Reply:
I don't think you can convince Spring to deal with different names. But would be fine for me if we go for `defaultLocal` (as it is)